clang/lib/CodeGen/CGCall.h

```diff
@@ -104,10 +109,13 @@ class CGCallee {
 
   /// Construct a callee.  Call this constructor directly when this
   /// isn't a direct call.
-  CGCallee(const CGCalleeInfo &abstractInfo, llvm::Value *functionPtr)
+  CGCallee(
+      const CGCalleeInfo &abstractInfo, llvm::Value *functionPtr,
+      const CGPointerAuthInfo &pointerAuthInfo = /*FIXME*/ CGPointerAuthInfo())
```

Comment:
Could you please provide additional context for this FIXME?

Reply:
The `CGPointerAuthInfo` parameter is going to be mandatory later.

Reply:
Yes, that's correct.